### PR TITLE
[FLOWS-1490] - Search field improvements

### PR DESCRIPTION
**Pull request by acnormun**
### Type of Change
* * [x] Bugfix
* * [ ] Feature
* * [ ] Code style update (formatting, local variables)
* * [ ] Refactoring (no functional changes, no api changes)
* * [ ] Tests
* * [ ] Other

### Motivation and Context
Removal of the browser default search card and add navigation through cards using the enter key.

### Summary of Changes
Added prevent default to Ctrl+F and focus in text input, and a keydown listener to search card.

### CODE REVIEW

**Review by Aldemylla — CHANGES_REQUESTED** [review]

File: src/components/search/SearchBar.tsx [hunk 1]
```diff
@@ -37,8 +37,20 @@ export interface SearchStoreProps {
 }
 
 export class SearchBar extends React.PureComponent<SearchStoreProps, {}> {
+  public componentDidMount(): void {
+    document.addEventListener('keydown', this.handleWindowKeyDown);
+  }
+
+  private handleWindowKeyDown(event: any): void {
+    if (event.key === 'Enter') {
+      const down = document.querySelector('#buttons button') as HTMLButtonElement;
```
Comment (paulobernardoaf) [inline src/components/search/SearchBar.tsx:46]: It's better to use a ref here instead of query selector

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "@weni/flow-editor",
   "license": "AGPL-3.0",
   "repository": "git://github.com/weni-ai/floweditor.git",
-  "version": "2.12.0-alpha.21",
+  "version": "2.12.0-alpha.22",
```
Comment (paulobernardoaf) [inline package.json:5]: Remember to remove any alpha references before merging it into the main branch.

File: src/components/canvas/Canvas.tsx [hunk 3]
```diff
@@ -440,16 +440,19 @@ export class Canvas extends React.PureComponent<CanvasProps, CanvasState> {
...
+    if (event.key.toLowerCase() === 'f' && (event.ctrlKey || event.metaKey)) {
+      event.preventDefault();
       this.props.handleSearchChange({
         isSearchOpen: true,
         nodes: [],
         value: '',
         selected: 0
       });
+
+      const input = (document.getElementsByClassName(
```
Comment (paulobernardoaf) [inline src/components/canvas/Canvas.tsx:452]: Here is alto better to use a ref to make sure that the input element used is the search input.

File: src/components/search/SearchBar.tsx [hunk 4]
```diff
@@ -156,7 +168,7 @@ export class SearchBar extends React.PureComponent<SearchStoreProps, {}> {
             autocomplete={true}
           />
         </div>
-        <div className={styles.buttons}>
+        <div className={styles.buttons} id="buttons">
```
Comment (paulobernardoaf) [inline src/components/search/SearchBar.tsx:171]: This id becomes obsolete when switching to ref.

**Review by paulobernardoaf — CHANGES_REQUESTED** [review]

**Review by paulobernardoaf — APPROVED** [review]

**Pull request merged**